addons/web_editor/views/snippets.xml

```diff
@@ -113,6 +116,57 @@
                 <t t-set="indent" t-value="True"/>
             </t>
         </div>
+
+        <div data-js="BackgroundShape" string="Shape"
+             t-att-data-selector="selector"
+             t-att-data-exclude="exclude"
+             t-att-data-target="target">
+            <we-select-pager class="o_we_shape_menu" string="Shape" data-dependencies="!shape_none_opt" data-name="bg_shape_opt">
+                <!-- Invisible None button to be used for select's placeholder text -->
+                <we-button data-shape="" data-name="shape_none_opt"></we-button>
+                <we-select-page>
+                    <we-button data-shape="website/shape_bg_1" data-select-label="Shape 1"></we-button>
```

Comment:
XML with mention of website inside web_editor... :wink: + the JS option is defined in web_editor but the related css for it to work is defined in website... :wink: